lib/hydranorth/preservation_queue.rb

```diff
@@ -0,0 +1,17 @@
+module Hydranorth::PreservationQueue
+
+  QUEUE_NAME = YAML.load(File.read('config/preservation.yml'))[Rails.env]['queue_name']
+
+  def queue
+    $queue ||= ConnectionPool.new({size: 1, timeout: 5}) { Redis.new }
```

Comment:
Yeah, that configures the redis connection. We could add the Connection Pool stuff, though.